src/components/Home.jsx

```diff
@@ -1,7 +1,43 @@
+import { useContext, useState } from "react";
+import { UserContext } from "./contexts/User";
+
 export function Home() {
+    const [selectedUser, setSelectedUser] = useState('')
+    const {user, setUser} = useContext(UserContext)
+
+    const users = ['cooljmessy', 'grumpy19', 'happyamy2016', 'jessjelly', 'tickle122', 'weegembump']
+
```

Comment:
Hard coded is alright here, but you may also want to make it dynamic by making the api call to get all users in case the app is going to have more functionality in future

Reply:
There is no way to POST a new user in my backend so i won't be able to add new users for now